mod-source-record-manager-server/src/main/java/org/folio/services/JobExecutionServiceImpl.java

```diff
@@ -240,7 +242,9 @@
     String userId = dto.getUserId();
     if (dto.getSourceType().equals(InitJobExecutionsRqDto.SourceType.ONLINE)) {
       JobProfileInfo jobProfileInfo = dto.getJobProfileInfo();
-      jobProfileInfo.withName(DEFAULT_JOB_PROFILE);
+      if (jobProfileInfo.getId().equals(DEFAULT_JOB_PROFILE_ID)) {
```

Comment:
you can add this condition to the one above

Reply:
@KaterynaSenchenko  yes, but in that case(which you described), 248-250 lines won`t reachable if it will be only ONLINE SourceType. Is it acceptable for us?

Reply:
Oh, I see, disregard the comment then